### ISSUE TITLE
please submit this to SBo.

### ISSUE THREAD

**Issue by adcdam** [ra3xdh/QucsS.SlackBuild#2]
well this is not a issue, but it would be nice if you submit this to slackbuilds.org.
best regards!

**Comment by ra3xdh** [comment]
Yes, I will consider the SlackBuild submission in the near future.

**Comment by adcdam** [comment]
hi i i took the liberty to modify the slackbuild to SBo standard (man pages goes in /usr/man not /usr/share),  fix the info file, fix the slack-desc and making a README and a doinst.sh , this way the slackbuild can be submitted here  https://slackbuilds.org/submit/
please submit the file in best case its approved and available for everyone in SBo.
best regards!
https://www.mediafire.com/file/8ihu0q0pp7y1fx9/qucs-s.tar.gz/file
